packs/data/spell-effects.db/spell-effect-blessing-of-defiance.json

```
@@ -1,10 +1,10 @@
 {
-    "_id": "FD9Ce5pqcZYstcMI",
+    "_id": "w9HpSwBLByNyRXvi",
```

```suggestion
    "_id": "FD9Ce5pqcZYstcMI",
```
As Simon says (hah) we don't want ID changes if we can help it.